packages/flutter/test/material/raw_material_button_test.dart

```diff
@@ -1,3 +1,7 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
```

Comment:
Technically, I think the year to use is the one in which this file was originally authored, not the day the copyright notice was added.  You can find this out in the git log for the file.